src/packages/math/src/matrix4.js

```diff
@@ -28,6 +28,18 @@ import vec4_transformMat4 from 'gl-vec4/transformMat4';
 
 const IDENTITY = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1];
 
+export function validateMatrix4(m) {
+  return m.length === 16 &&
+    Number.isFinite(m[0]) && Number.isFinite(m[1]) &&
```

Comment:
> Why not just m.every(Number.isFinite)?

This is old code that I just moved around. Pretty sure this is due to a premature assumption about speed, that inline comparisons are than a loop of function calls,.

We already know that this math library has a performance issue with class creation and it really needs a benchmark suite - out of scope for this PR.